rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -42,7 +42,7 @@ then
     fi
   fi
 fi
-if [ "${CFRUDDER_SERVER}" != "" ]
+if [ ! -f /var/rudder/cfengine-community/policy_server.dat ] && [ "${CFRUDDER_SERVER}" != "" ]
```

Comment:
I think this if should go to the script caller.
If the caller passes a server argument it should be applied